src/bokeh/command/subcommands/settings.py

```diff
@@ -119,39 +138,67 @@ class Settings(Subcommand):
             help="Show detailed help for a specific setting",
         )),
 
-        ('setting_name', Argument(
-            nargs='?',
-            help="Name of a specific setting to show detailed help for (use with -v)",
+        ('setting_names', Argument(
+            nargs='*',
+            help="One or more specific settings to show info for (use with -v for details)",
         )),
 
     )
 
     def invoke(self, args: Namespace) -> None:
-        '''
-
+        ''' Handle the "bokeh settings" command behavior.
         '''
         all_settings = get_all_settings()
 
-        if args.setting_name:
-            if args.setting_name in all_settings:
-                if args.verbose:
-                    self._print_setting_detail(args.setting_name, all_settings[args.setting_name])
-                else:
-                    print("To get detailed help for a specific setting, use:")
-                    print("  bokeh settings [-v | --verbose] <setting_name>")
-                    print("\nFor a list of all settings, use:")
-                    print("  bokeh settings")
-            else:
-                print(f"Setting '{args.setting_name}' not found.")
-                print("Available settings:")
-                for name in sorted(all_settings):
-                    print(f"  {name}")
+        if args.verbose and not args.setting_names:
+            for name, descriptor in all_settings.items():
+                self._print_setting_detail(name, descriptor)
+
+        elif args.setting_names:
+            resolved = resolve_setting_names(args.setting_names, all_settings)
+            self._print_resolved_settings(resolved, all_settings, args.verbose)
+
         else:
             self._print_settings_table(all_settings)
 
+
+    def _print_resolved_settings(
+        self, resolved: ResolutionResult, all_settings: dict[str, PrioritizedSetting[Any]], verbose: bool,
+    ) -> None:
+        """Print results from resolve_setting_names()."""
+
+
+        # Fuzzy matches
+        for name, close in resolved.fuzzy_matches.items():
+            print()
+            print(f"Setting '{name}' not found.")
+            print("Did you mean one of these?")
+            for c in close:
+                print(f"  {c}")
+            print()
+
+        # Not found
+        for name in resolved.not_found:
+            print()
+            print(f"Setting '{name}' not found.")
+            print()
+            print("Available settings:")
+            for n in sorted(all_settings):
+                print(f"  {n}")
+            print()
```

Comment:
It might be a little overkill, but since already require Jinja as a dependency, could consider using Jinja templates instead of these multi-prints-in-a-row. I leave that to your discretion in case it's something you'd like to explore. But given Jinja's conditional options, I think e.g. a single temple could handle both the "basic" and "detail" cases, simplifying the actual code needed.